core/math/bvh_debug.inc

```diff
@@ -22,7 +22,7 @@ String _debug_aabb_to_string(const BVHABB_CLASS &aabb) const {
 	sz += itos(-aabb.neg_max.z);
 	sz += ") ";
 
-	Vector3 size = aabb.calculate_size();
+	Point size = aabb.calculate_size();
 	float vol = size.x * size.y * size.z;
```

Comment:
Presumably this bit below won't work either for 2d, as there is no `size.z`. This function is likely not compiled in and only used for debugging .. it's not absolutely necessary to have it 2d / 3d aware.

Reply:
Good catch! It shouldn't be too hard to fix, just for the sake of having a 2D version of the debug stuff.